src/containers/views/Configuration.js

```diff
@@ -61,16 +72,33 @@ export class Configuration extends Component {
         {errors && errors.length > 0 && <Errors errors={errors} />}
         <div className="content-header">
           <h1>Configuration</h1>
-          <div className="page-buttons">
+          <div className="page-buttons multiple">
+            <Button // TODO: Hide toggle for non-YAML config files (e.g. '_config.toml')
+              onClick={this.toggleView.bind(this)}
+              type="view-toggle"
+              active={true}
+              triggered={this.state.guiView}
+              block />
             <Button
               onClick={this.handleClickSave}
               type="save"
-              active={editorChanged}
-              triggered={updated} />
+              active={editorChanged || fieldChanged}
+              triggered={updated}
+              block />
           </div>
         </div>
+          {
+            this.state.guiView && content &&
+              <div className="content-body">
+                <div className="warning">
+                  CAUTION! Any existing comments and formatting will be lost when editing via this view.
```

Comment:
:+1:  but should it be covered in a separate PR to discuss the translated versions?